### DISCUSSION TITLE
Soft freeze on merging contracts-bedrock PRs to develop for Holocene release

### DISCUSSION THREAD

**Discussion by smartcontracts** [ethereum-optimism/optimism#12725] [Announcements]
Hi all, please note that we are enforcing a ~1 week soft freeze on merging any PRs that change files within `packages/contracts-bedrock` for the sake of being able to easily merge the Holocene release branch back into `develop`.

Some conditions on this freeze:
- Net new files are ok
- Changes to existing files should not be merged
- Critical changes that must be made might be allowed to be merged if an exception is made

Freeze will be lifted approx ~1 week from the creation of this post. Ty!

**Comment by maurelian** [comment]
I definitely appreciate the patience extended by others here. For context the PR I'm working on that has caused me to ask for the freeze [is here](https://github.com/ethereum-optimism/optimism/pull/12746/files#diff-2966ca14703fa1322817c465794f157ec6fc1dc62d3a253dc5b455828b773bd5). It affects many of the core and bridge contracts on both L1 and L2, as well as associated tests, deploy scripts and L2 genesis generation. Efforts were made to break it up into smaller pieces but the nature of the changes makes this difficult. 

The freeze was requested in order to prevent a large rebase, which has taken several hours on previous occasions.

If you have a PR which you do not feel will conflict significantly with #12746, please feel free to ping me and I'm happy to consider waiving it. 

I currently expect to have this PR in by no later than November 6th.

**Comment by maurelian** [comment]
Closing the freeze.
